keyboards/db/db63/keymaps/default/keymap.c

```diff
@@ -0,0 +1,63 @@
+    [0] = LAYOUT_63_ansi(
+        KC_GESC, KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS,KC_EQL, KC_BSPC,
+        KC_TAB, KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC,KC_RBRC,KC_BSLS,
+        MO(3),  KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,KC_ENT,
+        KC_LSFT,KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,     KC_RSFT  ,   KC_UP ,MO(2),
+        KC_LCTL,KC_LALT,KC_LGUI,                KC_SPC,                   KC_SLSH,MO(1), KC_LEFT, KC_DOWN,KC_RIGHT
+    ),
+    [1] = LAYOUT_63_ansi(
+        KC_GESC, KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS,KC_EQL, KC_BSPC,
+        KC_TAB, KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC,KC_RBRC,KC_BSLS,
+        KC_CAPS,  KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,_______,
+        KC_LSFT,KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,     KC_RSFT  ,   KC_UP ,XXXXXXX,
+        KC_LCTL,KC_LALT,KC_LGUI,                KC_SPC,                   _______,MO(1), KC_LEFT, KC_DOWN,KC_RIGHT
+    ),
+    [2] = LAYOUT_63_ansi(
+        KC_GESC, RGB_MODE_PLAIN,   RGB_MODE_BREATHE,   RGB_MODE_RAINBOW,   RGB_MODE_SWIRL,  RGB_MODE_SNAKE,  RGB_MODE_KNIGHT,  RGB_MODE_XMAS,  RGB_MODE_GRADIENT,  RGB_MODE_RGBTEST,   RGB_TOG,  RGB_VAD,   RGB_VAI, KC_BSPC,
+        KC_TAB, KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC,KC_RBRC,KC_BSLS,
+        XXXXXXX,  KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,_______,
+        KC_LSFT,KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,     KC_RSFT  ,   KC_UP ,MO(2),
+        KC_LCTL,KC_LALT,KC_LGUI,                KC_SPC,                   _______,XXXXXXX, KC_LEFT, KC_DOWN,KC_RIGHT
+    ),
+    [3] = LAYOUT_63_ansi(
+        _______, _______,_______, _______,_______,_______,_______,_______,BL_STEP,BL_BRTG,BL_TOGG,BL_DEC,BL_INC,_______,
+        KC_TAB, KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC,KC_RBRC,KC_BSLS,
+        MO(3),  KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,RESET,
+        KC_LSFT,KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,     KC_RSFT  ,   KC_UP ,XXXXXXX,
+        KC_LCTL,KC_LALT,KC_LGUI,                KC_SPC,                   _______,XXXXXXX, KC_LEFT, KC_DOWN,KC_RIGHT
+    ),
+    /* You can copy this layer as base for a new fn layer * /
+    ), // */
+};
+
+void led_set_user(uint8_t usb_led) {
+    if (usb_led & (1<<USB_LED_CAPS_LOCK)) {
+        rgblight_sethsv(HSV_CYAN);
+    } else if (isrecorded) {
+        rgblight_sethsv(HSV_SPRINGGREEN);
+    } else {
+        rgblight_sethsv(HSV_GREEN);
+    }
+}
```

Comment:
Per the above, this would be instead:
```c
bool led_update_user(led_t led_state) {
    if (led_state.caps_lock) {
        rgblight_sethsv(HSV_CYAN);
    } else {
        rgblight_sethsv(HSV_GREEN);
    }
}
```